src/precompile.jl

```diff
@@ -118,6 +118,7 @@ let
                     Pkg.add(Pkg.PackageSpec(path = "TestPkg.jl/"))
                     Pkg.update(; update_registry = false)
                     Pkg.status()
+                    Pkg.Registry.update()
```

Comment:
Aren't we intentionally avoiding any internet activity? Or at least trying to i.e. `Pkg.UPDATED_REGISTRY_THIS_SESSION[] = true` above.

I think there's an issue open but can't find it.

Reply:
I thought that since we already install a local registry it wouldn't go out to the internet to update anything.

Reply:
huh. Ok. I guess we can delete `Pkg.UPDATED_REGISTRY_THIS_SESSION[] = true` and the `update_registry = false` kwarg on `Pkg.update`. Worth checking offline precompilation doesn't error.

Reply:
I did get

```
┌ Warning: could not download https://pkg.julialang.org/registries
│   exception = RequestError: Could not resolve host: pkg.julialang.org while requesting https://pkg.julialang.org/registries
└ @ Pkg.Registry ~/JuliaPkgs/Pkg.jl/src/Registry/Registry.jl:77
```

without internet. With the explicit `precompile` for updating registries it seems it wasn't super important so I took it away.